fetch.bs

```diff
@@ -1519,6 +1519,14 @@ user-agent-defined object). Unless otherwise stated it is null.
 <a lt=fetch for=/>fetching</a>. It provides a convenient way for standards to not have to set
 <a for=/>request</a>'s <a for=request>origin</a>.
 
+<p>A <a for=/>request</a> has an associated <dfn export for=request
```

Comment:
Unlike HTML, Fetch doesn't wrap inside phrasing elements, including `dfn` and `a`. I can take care of this though.

Also, I guess we have to wait a bit with landing this until clone is indexed, so prolly tomorrow.

Reply:
Fixed the wrapping. Let's wait for the specref!